core/capabilities/webapi/trigger/trigger.go

```diff
@@ -0,0 +1,299 @@
+package webapi
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+
+	ethCommon "github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/types/core"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/api"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/connector"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/common"
+	"github.com/smartcontractkit/chainlink/v2/core/services/gateway/handlers/webapicapabilities"
+)
+
+const defaultSendChannelBufferSize = 1000
+
+var webapiTriggerInfo = capabilities.MustNewCapabilityInfo(
+	webapicapabilities.TriggerType,
+	capabilities.CapabilityTypeTrigger,
+	"A trigger to start workflow execution from a web api call",
+)
+
+type Input struct {
+}
+type TriggerConfig struct {
+	AllowedSenders []string `toml:"allowedSenders"`
+	AllowedTopics  []string `toml:"allowedTopics"`
+	// RateLimiter    common.RateLimiterConfig `toml:"rateLimiter"`
+	RateLimiter    *values.Map `toml:"rateLimiter"`
```

Comment:
I saw that the `float` support was merged in for `values` library. Does this need to be `*values.Map` still? or can we just use `common.RateLimiterConfig` here and have `ValidateConfig` unwrap to the struct directly

Reply:
Yes, that now works!  Thx for the callout.